packages/ui-components/src/components/form/select/BadgeSelected.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <FormSelectBase
+    v-model="selectedValue"
+    :multiple="multiple"
+    :items="items"
+    :label="label"
+    :name="name"
+    :help="help"
+    :rules="rules"
+  >
+    <template #something-selected="{ value }">
+      <ul class="flex flex-wrap gap-1.5 text-xs">
+        <li v-for="item in isArrayValue(value) ? value : [value]" :key="item">
```

Comment:
Item can be an object, I don't think it can be a proper key. Additionally, you aren't passing in a `by` prop to `FormSelectBase` which is used to get the actual ID of each item (see any other selector that uses FormSelectBase), so equality checks might not work at times.

The `by` field specifies which property can you find the key/id of each item in. Considering that we allow any kind of structure for items, we should make `by` a prop and allow devs to feed in their own values.

Reply:
I think I have got this one done correctly, but I won't mark it as resolved until you have checked it just incase I did something wrong. 

I added a prop to BadgeSelected component for by and set the key of the v-for to :key="item[by]"

In FE2, I added by="id"